packages/animation/src/withAnimator/withAnimator.test.tsx

```diff
@@ -183,13 +183,13 @@ test('Should allow passing a "ref" to wrapped component', () => {
       return 100;
     }
   }
-  const ExampleNode = withAnimator()(ExampleComponent);
+  const ExampleNode = withAnimator<ExampleComponent>()(ExampleComponent);
 
   const ExampleApp: FC = () => {
-    const ref = useRef<any>();
+    const ref = useRef<ExampleComponent>(null);
```

Comment:
Seems the typings for react are a bit off here. If useRef is not initialized, it returns `React.MutableRefObject<ExampleComponent | undefined>`, where as if its initialized with null it returns: `React.RefObject<ExampleComponent>`.

Reply:
I guess it's alright.